pytorch_lightning/plugins/apex.py

```diff
@@ -98,3 +101,38 @@ def configure_apex(self, amp, model, optimizers, amp_level):
         """
         model, optimizers = amp.initialize(model, optimizers, opt_level=amp_level)
         return model, optimizers
+
+    def clip_gradients(self, grad_clip_val, optimizer):
```

Comment:
```suggestion
    def _clip_gradients(self, grad_clip_val, optimizer):
```
it shall be parent overwrite here, right?

Reply:
This doesn't inherit from the accelerator class, but is called within the plugin. Let's do some renaming since there is a lot of confusion here.

Reply:
After thinking through I don't think it's an easy rename any suggestions @Borda? otherwise I'll merge and revise in a separate PR when needed.

Reply:
not sure, maybe add some docstring with an explanation...